MailTests/Folders/FolderStructureGenerator.swift

```diff
@@ -46,7 +46,14 @@ public struct FolderStructureGenerator {
             Recipient.self,
             Draft.self,
             Signature.self,
-            SearchHistory.self
+            SearchHistory.self,
```

Comment:
maybe I can import a configuration from the main app, in order to not have to maintain it test side.